pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/model/conditions/Aborted.groovy

```diff
@@ -38,7 +38,9 @@ import org.jenkinsci.plugins.workflow.job.WorkflowRun
 public class Aborted extends BuildCondition {
     @Override
     public boolean meetsCondition(WorkflowRun r) {
-        return r.getResult() != null && r.getResult().equals(Result.ABORTED)
+        Result execResult = getExecutionResult(r)
+        return (execResult != null && execResult.equals(Result.ABORTED)) ||
```

Comment:
You could clean this up a bit to make it more readable. Groovy performs a `.equals` bu default with the `==` operator. But that doesn't really matter since they should all be the same instance anyway, so an `==` operator should catch it in standard java as well.

```
return execResult == Result.ABORTED || r.getResult() == Result.ABORTED
```

Reply:
Oh right, null safety for free. =)